packages/coordinator/src/services/communication/sandbox-spine-client.ts

```diff
@@ -32,15 +29,14 @@ export class SandboxSpineClient implements SpineClient {
   }
 
   async handleTrackerRequest(): Promise<spine.SpineResponse<unknown>> {
-    return Promise.resolve({
-      statusCode: 400,
-      body: notSupportedOperationOutcome
-    })
+    return notSupportedOperationOutcomePromise()
   }
 
   async handleSpineRequest(spineRequest: spine.SpineRequest): Promise<spine.SpineResponse<unknown>> {
     switch (spineRequest.interactionId) {
       case hl7V3.Hl7InteractionIdentifier.PARENT_PRESCRIPTION_URGENT._attributes.extension:
+      case hl7V3.Hl7InteractionIdentifier.DISPENSE_NOTIFICATION._attributes.extension:
```

Comment:
This looks like we are changing the intent of the logic - we need to check this is correct as per business rules